### PR TITLE
keepass-plugin-readable-passphrase: Add version 1.2.1

### PR DESCRIPTION
**Pull request by rminderhoud**

### CODE REVIEW

File: bucket/keepass-plugin-readable-passphrase.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-1.2.1/ReadablePassphrase.1.2.1.plgx",
+    "hash": "d91261ae0001fd1c78775e8d4354b69bb4b33432811e8e66a4ba19eb185ac35a",
+    "installer": {
+        "script": "Copy-Item \"$dir\\$fname\" \"$(appdir keepass $global)\\current\\Plugins\""
+    },
+    "uninstaller": {
+        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\$fname\""
+    },
+    "checkver": {
+        "github": "https://github.com/ligos/readablepassphrasegenerator"
```
Comment (Ash258) [inline bucket/keepass-plugin-readable-passphrase.json:16]:
```suggestion
        "github": "https://github.com/ligos/readablepassphrasegenerator",
        "regex": "tag/release-([\\d.]+)"
```

File: bucket/keepass-plugin-readable-passphrase.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    "autoupdate": {
+        "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-$version/ReadablePassphrase.$version.plgx"
+    }
+}
```
Comment (Ash258) [inline bucket/keepass-plugin-readable-passphrase.json:21]:
```suggestion
}

```

File: bucket/keepass-plugin-readable-passphrase.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        "script": "Copy-Item \"$dir\\$fname\" \"$(appdir keepass $global)\\current\\Plugins\""
+    },
+    "uninstaller": {
+        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\$fname\""
+    },
+    "checkver": {
+        "github": "https://github.com/ligos/readablepassphrasegenerator"
+    },
+    "autoupdate": {
+        "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-$version/ReadablePassphrase.$version.plgx"
```
Comment (Ash258) [inline bucket/keepass-plugin-readable-passphrase.json:19]:
```suggestion
        "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-$version/ReadablePassphrase.$version.plgx#/ReadablePassphrase.plgx"

```

File: bucket/keepass-plugin-readable-passphrase.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "homepage": "https://github.com/ligos/readablepassphrasegenerator",
+    "license": "Apache-2.0",
+    "depends": "extras/keepass",
+    "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-1.2.1/ReadablePassphrase.1.2.1.plgx",
+    "hash": "d91261ae0001fd1c78775e8d4354b69bb4b33432811e8e66a4ba19eb185ac35a",
+    "installer": {
+        "script": "Copy-Item \"$dir\\$fname\" \"$(appdir keepass $global)\\current\\Plugins\""
+    },
+    "uninstaller": {
+        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\$fname\""
```
Comment (Ash258) [inline bucket/keepass-plugin-readable-passphrase.json:13]:
```suggestion
        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\ReadablePassphrase.plgx\""
```

File: bucket/keepass-plugin-readable-passphrase.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.2.1",
+    "description": "Plugin for KeePass 2.x that generates passphrases which are (mostly) grammatically correct but nonsensical.",
+    "homepage": "https://github.com/ligos/readablepassphrasegenerator",
+    "license": "Apache-2.0",
+    "depends": "extras/keepass",
+    "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-1.2.1/ReadablePassphrase.1.2.1.plgx",
```
Comment (Ash258) [inline bucket/keepass-plugin-readable-passphrase.json:7]:
```suggestion
    "url": "https://github.com/ligos/readablepassphrasegenerator/releases/download/release-1.2.1/ReadablePassphrase.1.2.1.plgx#/ReadablePassphrase.plgx",
```

File: bucket/keepass-plugin-readable-passphrase.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.2.1",
+    "description": "Plugin for KeePass 2.x that generates passphrases which are (mostly) grammatically correct but nonsensical.",
```
Comment (Ash258) [inline bucket/keepass-plugin-readable-passphrase.json:3]:
```suggestion
    "description": "KeePass 2.x plugin, that generates grammatically correct but nonsensical passphrases.",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by rminderhoud** [comment]
Thanks for the review @Ash258! The # at the end of the URL is a nice trick. I've made all the changes as requested.

File: bucket/keepass-plugin-readable-passphrase.json (hunk 3 above)
Comment (Ash258) [reply bucket/keepass-plugin-readable-passphrase.json:20]: !

Comment (rminderhoud) [reply bucket/keepass-plugin-readable-passphrase.json]: Whoops! Missed that URL. Good spotting. I've corrected it in my latest commit.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**